code/game/objects/items/defib.dm

```diff
@@ -132,13 +141,16 @@
 			M.putItemFromInventoryInHandIfPossible(src, H.held_index)
 
 /obj/item/defibrillator/screwdriver_act(mob/living/user, obj/item/tool)
-	if(cell)
-		cell.update_appearance()
-		cell.forceMove(get_turf(src))
-		cell = null
-		tool.play_tool_sound(src, 50)
-		to_chat(user, span_notice("You remove the cell from [src]."))
-		update_power()
+	if(!cell || !cell_removable)
+		return FALSE
+
+	cell.update_appearance()
+	cell.forceMove(get_turf(src))
+	cell = null
+	tool.play_tool_sound(src, 50)
+	to_chat(user, span_notice("You remove the cell from [src]."))
```

Comment:
Move this up a hair so you can say" you remove [cell] from [src]"
Or just balloon alert "removed [cell]"

Reply:
Sure, I love balloon alerts for feedback messages like this. Changed to a balloon alert.